physics/gcycle.F90

```diff
@@ -75,6 +75,7 @@ subroutine gcycle (me, nthrds, nx, ny, isc, jsc, nsst, tile_num, nlunit, fn_nml,
                                            slmsk(:)
     integer,              intent(inout) :: vtype(:),   &
                                            stype(:),   &
+                                           scolor(:),  &        !soil color
```

Comment:
Same comment RE: the `!soil color` comments. A developer should be able to trace the new variable to the metadata where it is described.